File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q20.py
def main():
    #Input
    temperatureC = float(input("Enter a temperature valeu: "))

    #Processing
    temperatureF = (((9 * temperatureC) + 160)/5)

    #Output
    print(f"The temperature of {temperatureC} °C converted to °F is equal to: {temperatureF}")

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q20.js
import { question } from "readline-sync";
function main(){
    //Input
    const temperatureC = parseFloat(question("Enter a temperature in °C: ")) 

    //Processing
    const temperatureF = (9 * temperatureC + 160)/5

    //Output
    console.log(`The temperature of ${temperatureC} °C converted to °F is equal to: ${temperatureF} °F`)
}

main()